src/seedu/addressbook/AddressBook.java

```diff
@@ -420,7 +420,7 @@ private static String getMessageForSuccessfulAddPerson(String[] addedPerson) {
      * @return feedback display message for the operation result
      */
     private static String executeFindPersons(String commandArgs) {
-        final Set<String> keywords = extractKeywordsFromFindPersonArgs(commandArgs);
+        final Set<String> keywords = extractKeywordsFromFindPersonArgs(commandArgs.toLowerCase());
```

Comment:
Remember to include this in the comments so that it's clear that commands are case insensitive.